pipeline/data/importers.py

```diff
@@ -0,0 +1,221 @@
+import shutil
```

Comment:
Should this be called `download_parallel.py` instead so that it matches the `download_mono.py` and is less ambiguous?